zerver/lib/bugdown/__init__.py

```diff
@@ -560,12 +560,19 @@ def handleMatch(self, match):
 if settings.VOYAGER:
     path_to_emoji = os.path.join(os.path.dirname(__file__), '..', '..', '..',
                                  'prod-static', 'serve', 'third', 'gemoji', 'images', 'emoji', '*.png')
+    path_to_unicode_emoji = os.path.join(os.path.dirname(__file__), '..', '..', '..',
+                                         'prod-static', 'serve', 'third', 'gemoji', 'images',
+                                         'emoji', 'unicode', '*.png')
 else:
     path_to_emoji = os.path.join(os.path.dirname(__file__), '..', '..', '..',
                                  # This should be the root
                                  'static', 'third', 'gemoji', 'images', 'emoji', '*.png')
+    path_to_unicode_emoji = os.path.join(os.path.dirname(__file__), '..', '..', '..',
+                                         # This should be the root
+                                         'static', 'third', 'gemoji', 'images', 'emoji', 'unicode', '*.png')
```

Comment:
I think this is a good opportunity to clean this messy path code up.  We can instead write this as:
```
if settings.VOYAGER:
    static_path = 'prod-static/serve'
else: 
    static_path = 'static/'
emoji_tree = os.path.join(settings.DEPLOY_ROOT, static_path, "third", "gemoji", "images", "emoji")
path_to_emoji = os.path.join(emoji_tree, '*.png')
path_to_unicode_emoji = os.path.join(emoji_tree, '*.png')
```
Probably better yet would be to make use of `settings.STATIC_ROOT`, but this at least avoids having N copies of the `'..'` party...